React/Libraries.md

```diff
@@ -0,0 +1,53 @@
+- [react-datepicker](https://github.com/Hacker0x01/react-datepicker/) - A simple and reusable datepicker component for React
+- [react-virtualized](https://github.com/bvaughn/react-virtualized) - React components for efficiently rendering large lists and tabular data
+- [@tippyjs/react](https://github.com/atomiks/tippyjs-react) - The complete tooltip, popover, dropdown, and menu solution for the web, powered by Popper
+- [downshift](https://github.com/downshift-js/downshift) - A set of primitives to build autocomplete, combobox or select dropdown React components
+- [react-collapse](https://github.com/nkbt/react-collapse) - Component-wrapper for collapse animation with react-motion for elements with variable (and dynamic) height
+- [react-markdown](https://github.com/remarkjs/react-markdown) - Markdown component for React
+- [react-slider](https://github.com/zillow/react-slider) - Accessible, CSS agnostic, slider component for React
+- [react-qr-code](https://github.com/rosskhanas/react-qr-code) - A QR code generator for React and React Native
+- [react-modal](https://github.com/reactjs/react-modal) - Accessible modal dialog component for React
+- [react-cool-inview](https://github.com/wellyshen/react-cool-inview) - React hook to monitor an element enters or leaves the viewport (or another element)
+- [react-avatar-editor](https://github.com/mosch/react-avatar-editor) - Small avatar & profile picture component that allows resizing and cropping uploaded images using a intuitive user interface
+
+##### React Testing
+- [jest](https://github.com/facebook/jest) - Delightful JavaScript Testing
+- [react-testing-library](https://github.com/testing-library/react-testing-library) - Simple and complete React DOM testing utilities that encourage good testing practices
+- [react-hooks-testing-library](https://github.com/testing-library/react-hooks-testing-library) - Simple and complete React hooks testing utilities that encourage good testing practices
+- [@testing-library/user-event](https://github.com/testing-library/user-event) - Simulate user events for react-testing-library
+
+##### React Libraries
+- [react-transition-group](https://github.com/reactjs/react-transition-group) - An easy way to perform animations when a React component enters or leaves the DOM
+- [react-focus-lock](https://github.com/theKashey/react-focus-lock) - Focus locking
+- [react-portal-universal](https://github.com/jesstelford/react-portal-universal) - Generic wrapper for React's createPortal allowing for rendering portals on the server
+
+##### React State Management
+- [mobx](https://github.com/mobxjs/mobx) - Simple, scalable state management
+- [datx](https://github.com/infinum/datx) - A MobX data store
+
+##### Forms
+- [react-hook-form](https://github.com/react-hook-form/react-hook-form) - React Hooks for forms validation (Web + React Native)
+- [formik](https://github.com/formium/formik) - Build forms in React, without the tears
```

Comment:
we should use this anymore

Reply:
I agree. I think we have a few older projects that still use it. Removing it either way?

Reply:
Remove Formik definitely!

Reply:
Removed